src/txn/txn_recover.c

```diff
@@ -1152,7 +1153,7 @@ __wt_txn_recover(WT_SESSION_IMPL *session, const char *cfg[], bool disagg)
      * 2. The history store file was found in the metadata.
      * 3. We are not using disaggregated storage.
      */
-    if (hs_exists && !F_ISSET(conn, WT_CONN_READONLY) && !disagg) {
+    if (hs_exists_local && !F_ISSET(conn, WT_CONN_READONLY) && !disagg) {
```

Comment:
What's the recovery story for the shared history store? This code ignores it, which I think makes sense because it's shared, but I want to confirm that.

Reply:
I see - thanks for the explanation. Given the new information we got this morning on local tables I'm comfortable believing this won't be a problem.